db/backends/postgresql/schema.py

```diff
@@ -8,6 +8,8 @@ class DatabaseSchemaEditor(PostgresDatabaseSchemaEditor):
     ON_DELETE_DEFAULT = 'NO ACTION'
     ON_UPDATE_DEFAULT = 'NO ACTION'
 
+    sql_create_index_concurrently = 'CREATE INDEX CONCURRENTLY %(name)s ON %(table)s (%(columns)s)%(extra)s'
```

Comment:
Just in case we ever want a concurrent GIN index or something, this should probably preserve the `%(using)s` in `sql_create_index`:
```
"CREATE INDEX CONCURRENTLY %(name)s ON %(table)s%(using)s (%(columns)s)%(extra)s"
```